.circleci/config.yml

```diff
@@ -37,15 +37,43 @@ jobs:
       - run:
           name: Run regular tests
           command: |
-            CLASSNAMES=$(circleci tests glob "**/src/test/**/**.java" | grep -v 'spring-cloud-kubernetes-integration-tests' \
```

Comment:
use `xargs grep -L '@Nested'` to find test classes that do _not_ have `@Nested` inside them.